README.md

```diff
@@ -1,8 +1,54 @@
-# Aurora project #
+# Table of Contents
+- [Introduction](#introduction)
+- [How to run](#how-to-run)
+  * [Ethercat interface](#ethercat-interface)
+  * [teleop_deploy](#teleop_deploy)
+  * [docker_deploy](#docker_deploy)
+  * [configure_software](#configure_software)
+  * [install_software](#install_software)
+  * [install_python3](#install_python3)
+- [Development](#development)
+  * [Development Docker](#development-docker)
+- [Testing](#testing)
+  * [Test creation](#test-creation)
+  * [Testing with molecule_docker](#testing-with-molecule_docker)
+  * [Private docker images](#private-docker-images)
+  * [Testing with molecule_ec2](#testing-with-molecule_ec2)
+  * [Credentials](#credentials)
+  * [Automatic tests](#automatic-tests)
+  * [Testing on real hardware](#testing-on-real-hardware)
+- [Structure of files](#structure-of-files)
+  * [Roles](#roles)
+  * [Docker](#docker)
+  * [Installation](#installation)
+  * [Products](#products)
+  * [Common](#common)
+  * [Templating](#templating)
+  * [Dependencies](#dependencies)
+- [Playbooks](#playbooks)
+  * [Playbook creation](#playbook-creation)
+- [Inventories](#inventories)
+- [Molecule tests](#molecule-tests)
+  * [Docker tests](#docker-tests)
+  * [AWS EC2 tests](#aws-ec2-tests)
+- [Syntax and rules](#syntax-and-rules)
+- [Tutorial 1 desktop icon](#tutorial-1-desktop-icon)
 
-The main purpose of this subsystem is to unify one-liners approaches based on Ansible best practices. This one-liner is able to install Docker, download the specified image and create a new container for you. It will also create a desktop icon to start the container and launch the hand.
+# Introduction #
 
-## Before running the docker_deploy playbook ##
+Aurora is an installation automation tool using Ansible. It uses Molecule for testing Ansible scripts and it has automated builds in AWS EC2/CodeBuild and DockerHub. It can be used to develop, test and deploy complicated, multi-machine, multi-operating-system automated installs of software. Aurora's purpose is to unify one-liners approaches based on Ansible best practices.
+
+For example, it's possible to use Aurora to install Docker, download the specified image and create a new container for you. It will also create a desktop icon to start the container and launch the hand.
+
+Ansible user guide is available [here](https://docs.ansible.com/ansible/latest/user_guide/index.html) (Aurora is currently using Ansible 2.8.1)
+
+Molecule user guide is available [here](https://molecule.readthedocs.io/en/stable/) (Aurora is currently using Molecule 2.20.1)
+
+# How to run #
+
+## Ethercat interface ##
```

Comment:
this is not needed for teleop right? maybe it can be a subsection or hand_e and h products?

Reply:
fixed